{"name": "Negative test: Patch object with invalid data", "status": "skipped", "statusDetails": {"message": "Skipped: API does not support PATCH method (405)", "trace": "('C:\\\\Users\\\\Sergey\\\\Projects\\\\api-automation-framework\\\\tests\\\\test_patch_object_negative.py', 8, 'Skipped: API does not support PATCH method (405)')"}, "parameters": [{"name": "patch_data", "value": "{}"}], "start": 1748319071652, "stop": 1748319071652, "uuid": "4227e545-92b1-4126-b7b4-36f847144546", "historyId": "ab19a2598315c063e429f44853595384", "testCaseId": "70f805e9fced0522ce229ca93232757c", "fullName": "tests.test_patch_object_negative#test_patch_object_negative", "labels": [{"name": "tag", "value": "@pytest.mark.skip(reason='API does not support PATCH method (405)')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_patch_object_negative"}, {"name": "host", "value": "Sergo"}, {"name": "thread", "value": "15932-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_patch_object_negative"}]}